src/skyro/company/Main.java

```diff
@@ -3,6 +3,75 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        // Задание №1
+        int[] arr = new int[3];
+        arr[0] = 1;
+        arr[1] = 2;
+        arr[2] = 3;
+        for (int i = 0; i < 3; i++) {
+            if (i<2) {
+                System.out.print(arr[i] + ",");
```

Comment:
```suggestion
                System.out.print(arr[i] + ", ");
```
для лучшей читаемости